### ISSUE TITLE
Custom rules for Belote ?

### ISSUE THREAD

**Issue by marwendoukh** [Binary-Team/Rachma-Android#27]
Do we need to set custom rules for Belote ?
/cc @mohamedamineznaidi

**Comment by mohamedamineznaidi** [comment]
No, it's the same but in other contries the score taking may change

**Comment by marwendoukh** [comment]
So we need to give the user the ability to set the rules that works for him. Right ?
